packages/engine-api-client/test/unit/executionEngine/httpRetry.test.ts

```diff
@@ -45,18 +46,12 @@ describe("ExecutionEngine / http ", () => {
 
     baseUrl = await server.listen(0);
 
-    executionEngine = new ExecutionEngineHttp(
-      {
-        urls: [baseUrl],
-        retryAttempts: defaultExecutionEngineHttpOpts.retryAttempts,
-        retryDelay: defaultExecutionEngineHttpOpts.retryDelay,
-      },
-      {signal: controller.signal}
-    );
+    executionEngine = new ExecutionEngineHttp({urls: [baseUrl], retryAttempts}, {signal: controller.signal});
   });
 
   describe("notifyForkchoiceUpdate", function () {
     it("notifyForkchoiceUpdate no retry when no pay load attributes", async function () {
+      this.timeout("10 min");
```

Comment:
Why is the 10 minutes timeout necessary? Sounds very extreme, this is a unit test!

Reply:
Not sure why this is showing as a diff now. It is currently same on unstable
https://github.com/ChainSafe/lodestar/blob/70796d3ab8736e15551598fe44029b11b3741091/packages/beacon-node/test/unit/executionEngine/httpRetry.test.ts#L91

I'll change this to 60s